web_responsive/static/src/components/apps_menu/apps_menu.esm.js

```diff
@@ -31,7 +31,8 @@ patch(WebClient.prototype, {
                 ["is_redirect_home"]
             );
             this.env.services.user.updateContext({
-                is_redirect_to_home: is_redirect_home[0].is_redirect_home,
+                is_redirect_to_home:
+                    is_redirect_home[0] && is_redirect_home[0].is_redirect_home,
```

Comment:
In v17 you should be able to use js optional chaining :smile: !
```suggestion
                is_redirect_to_home: is_redirect_home?.[0].is_redirect_home,
```

Reply:
Oh! I didn't know about it :sweat_smile: Thanks!